rfcs/convergence/react-focus-management.md

```diff
@@ -0,0 +1,182 @@
+| findLast  | HTMLElement   |                          | Finds the last focusable element                    |
+| findAll   | HTMLElement[] | boolean matcher function | Finds all focusable elements that match a condition |
+
+#### useKeyboardNavigationGroup
+
+A hook that returns a set of data-* attributes to be spread to the container of a group of focusable elements that should support keyboard navigation
+
+The below table lists the configuration options that the hook should support:
+
+| Name          | Value                  | Description                                                                                               |
+| ------------- | ---------------------- | --------------------------------------------------------------------------------------------------------- |
+| axis          | vertical \| horizontal | Indicates which direction keyboard navigation should use                                                  |
+| enableHomeEnd | boolean                | Enables the use of `Home` and `End` keys to navigate to first and last elements of the group respectively |
+| circular      | boolean                | Indicates that navigation should not stop at first and last elements respectively                         |
+
+### Pros and Cons
+
+<!-- Enumerate the pros and cons of the proposal. Make sure to think about and be clear on the cons or drawbacks of this propsoal. If there are multiple proposals include this for each. -->
+
+#### Pros
+
+The current development pace of converged components means we will be able iterate quickly on this API since components under development such as `Menu`, `Accordion` and `Tooltip` will require these behaviours in the coming project cycles
+
+Internally ability-helpers leverages the [TreeWalker](https://developer.mozilla.org/en-US/docs/Web/API/TreeWalker), which is the recommended the DOM tree traversal method [according to the w3c](https://www.w3.org/TR/DOM-Level-2-Traversal-Range/traversal.html). This API is dynamic and operations are designed represent underlying changes to the DOM which is a big plus for handling dynamic or lazy loaded content through a rendering framework.
+
+A hook based implementation will be a lot easier to maintain for end users, since function calls can be added/removed/mocked easily which leads to easier adaptation in large codebases where extensive unit or snapshot testing is involved.
+
+#### Cons
+
+The library is new, and currently partially used in TMP. Not all issues have been identified. And contributions were still needed to the library to support the initial keyboard navigation API proposed in this RFC
+
+For backwards compat and support, the `FocusZone` components will still be needed for partners that don't use Function components in React, which are still relatively new.
+
+The complex bi-directional navigation `FocusZone` supports has not been explored with ability-helpers, and might require contributions back to the library to support such a case.
```

Comment:
+1 on this, we should really strive to bring contributions back to `tabster` that add this functionality.